Vineyard/Vineyard/Preview Content/VineBranchView.swift

```diff
@@ -0,0 +1,128 @@
+
+//
+//  VineBranchView.swift
+//  Vineyard
+//
+//  Created by Jiyoon Lee on 11/5/24.
+//
+
+import SwiftUI
+
+struct VineBranchView: View {
+    @State private var hasScrolledToEnd: Bool = false
+    @State private var hasScrolledToStart: Bool = true
+    var isStaticPreview: Bool = false
+    var body: some View {
+        ZStack {
+            if isStaticPreview {
+                staticImagePreview()
+            } else {
+                EndDetectionScrollView(.horizontal, showIndicators: false, hasScrolledToEnd: $hasScrolledToEnd, hasScrolledToStart: $hasScrolledToStart) {
+                    LazyHStack {
+                        ForEach(1..<11, id: \.self) { index in
+                            Image("panorama\(index)")
+                                .aspectRatio(contentMode: .fit)
+                                .containerRelativeFrame(.horizontal, count:2, spacing: 10.0)
+                            
+                        }
+                    }
+                }
+                .frame(height: 300)
+                .ignoresSafeArea()
+            }
+
+            if hasScrolledToEnd && !isStaticPreview {
+                arrowView(direction: .left)
+            }
+
+            if hasScrolledToStart && !isStaticPreview {
+                arrowView(direction: .right)
+            }
+        }
+    }
+
+    private func staticImagePreview() -> some View {
+        HStack(spacing: 0) {
+            //chose 1..<3 to check if images look connected
```

Comment:
what do these comments mean? I think I get the point of the static image preview but just wanna understand why its 3 specifically

Reply:
oh i wanted to show that two images look connected
but last two images which are picture of grapes, does not clearly show if two images look seamlessly connected or not
so i used panorama images

i used 3 cuz u wanted 2 images to show up